src/6model/reprs/MVMIter.c

```diff
@@ -105,12 +105,29 @@ static void shift(MVMThreadContext *tc, MVMSTable *st, MVMObject *root, void *da
             }
             return;
         case MVM_ITER_MODE_HASH:
+            ; /* Sigh, C99 won't let me put a declaration here. */
+            MVMStrHashTable *hashtable = &(((MVMHash *)target)->body.hashtable);
```

Comment:
My usual trick is to use a compound statement (wrap the "body" of the `case` in curlies).

Reply:
IIRC I didn't really want to change the indenting of a bunch of lines that I wasn't modifying.

And I still suspect that the C99 spec/grammar is wrong here - if you're going to allow decarations *almost* anywhere, make declarations into statements, and hence permit them everywhere. But that ship sailed (in 1999)

Reply:
I just put the `{` after the `:` and the closing one at the same level as the `case`. :-)

Reply:
And yeah, agree the spec is weird here.